acquia/acquia_cms
# ACMS-712: Fix Scheduled tab in admin is not working.

**Motivation**

The $dashboard_owner return a string value and thus the hasPermission() function throws an error.
The $dashboard_owner should be loaded before checking the permission.

Fixes #NNN
https://www.drupal.org/files/issues/2020-05-04/user_load_moderation_dashboard_updated_with_DI_for_beta_version_3048782.patch

**Proposed changes**

 This PR contains a patch which fix  the issue  related   to  asPermission() in moderation_dashboard module.

**Alternatives considered**

NA

**Testing steps**

- login with admin
- Click on admin 
- Click on Scheduled tab
- Verify you are not seeing any error and scheduled page works fine.

**Merge requirements**
